pom.xml

```diff
@@ -129,6 +129,13 @@
         <version>2.14</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <!-- Required for running with Java 9+ -->
+        <groupId>org.jboss.marshalling</groupId>
+        <artifactId>jboss-marshalling-river</artifactId>
+        <version>2.0.6.Final</version>
+        <scope>test</scope>
+      </dependency>
```

Comment:
To avoid https://issues.jenkins-ci.org/browse/JENKINS-51998 .

This is actually introduced workflow-support-plugin 3.0 and later.
workflow-support-plugin 3.0 targets Jenkins 2.121.1+.

Reply:
copyartifact already targets 2.121.1.